src/frontends/pytorch/src/op_table.cpp

```diff
@@ -304,7 +305,8 @@ OP_CONVERTER(translate_new_ones_fx);
 OP_CONVERTER(translate_new_zeros_fx);
 OP_CONVERTER(translate_ones_fx);
 OP_CONVERTER(translate_ones_like_fx);
-OP_CONVERTER(translate_reflection_pad_nd_fx);
+OP_CONVERTER(translate_reflection_pad_nd);
```

Comment:
Please move it up to the section of regular non-fx ops

Reply:
fixed